server/controllers/student.py

```diff
@@ -396,29 +395,28 @@ def group_respond(name):
         flash("You are not in a group")
     else:
         try:
-
             if action == "accept":
                 group.accept(current_user)
                 subject = "{0} has accepted the invitation to join your group".format(current_user.email)
                 body = "Your group for {0} now has {1} members".format(assignment.display_name,
                                                                        len(group.members))
-                group_action_email(group.members, subject, body)
+                members = [m.user.email for m in group.members]
+                send_emails(members, subject, body)
```

Comment:
We call `send_emails` at the bottom of each cases  - ideas for something better that's a bit more DRY?